pkg/node/node.go

```diff
@@ -270,11 +290,7 @@ func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey,
 		eventListener := listener.New(logger, swapBackend, postageContractAddress, priceOracleAddress)
 		b.listenerCloser = eventListener
 
-		batchService, err := batchservice.New(batchStore, logger, eventListener)
-		if err != nil {
-			return nil, err
-		}
-		err = batchService.Start()
+		_, err := batchservice.New(batchStore, logger, eventListener)
```

Comment:
why `_`

Reply:
we dont need the  object, the constuctor  starts the listener is enough

Reply:
Ok but what about shutdown?